README.md

```diff
@@ -1,3 +1,70 @@
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict.

Reply:
huh...I just synced with head.  Must've forgotten?

Reply:
Oh...Ok, I'll re-up it.  Canceling.

Reply:
Sounds good.

I wonder how much effort it would be to get travis to recognize and fail builds with merge conflicts... hmm.